app/controllers/comments_controller.rb

```diff
@@ -1,30 +1,29 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
```

Comment:
missing before_action to authorize comment policy. How are you ensuring a user is authorized to comment?